cirq-core/cirq/protocols/has_unitary_protocol_test.py

```diff
@@ -26,6 +26,12 @@ class No:
     assert not cirq.has_unitary(No())
 
 
+def test_fail_fast_measure():
+    q = cirq.NamedQubit('q0')
```

Comment:
Add a test for a circuit as well to check for https://github.com/quantumlib/Cirq/issues/4959#issuecomment-1034269813

Reply:
Like so?